### PR TITLE
chapter_8

### PR DESCRIPTION
**Pull request by azimfaiz**

### CODE REVIEW

**Comment by azimfaiz** [comment]
ready

File: app/assets/stylesheets/sessions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/sessions.scss]: remove empty file

File: config/routes.rb [hunk 2]
```diff
@@ -6,5 +6,8 @@
   get   "/about", to: "static_pages#about"
   get   "/contact", to: "static_pages#contact"
   get   "/signup", to: "users#new"
+  get   "/login", to: "sessions#new"
```
Comment (ngocvt-0484) [inline config/routes.rb]:
```suggestion
  get "/login", to: "sessions#new"
```

**Comment by azimfaiz** [comment]
ready

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,20 @@
+
+<% provide :title, "Log in" %>
```
Comment (saobv-2223) [inline app/views/sessions/new.html.erb]:
```suggestion
<% provide :title, "Log in" %>
```

**Comment by azimfaiz** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**